app/models/route.rb

```diff
@@ -3,4 +3,16 @@ class Route < ApplicationRecord
   has_many :bus_stops_routes
   has_many :bus_stops, through: :bus_stops_routes
   default_scope { order :number }
+
+  def stops_by_sequence(direction)
```

Comment:
I appreciate how you used a method here instead of shoving that query into the method below.